<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QDialog" name="Configuracion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuracion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icono.qrc">
    <normaloff>:/icono</normaloff>:/icono</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Servidor:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="inDireccion">
     <property name="text">
      <string>http://</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Puerto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="inPuerto">
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>8080</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(165, 213, 229);</string>
     </property>
     <property name="title">
      <string>Habilitar Cajas</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chkCaja1">
        <property name="text">
         <string>Caja 1</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja2">
        <property name="text">
         <string>Caja 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja3">
        <property name="text">
         <string>Caja 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkCaja4">
        <property name="text">
         <string>Caja 4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbrir">
        <property name="text">
         <string>Abrir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnGuardar">
        <property name="text">
         <string>Guardar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icono.qrc"/>
 </resources>
 <connections/>
</ui>
